### ISSUE TITLE
I must say, this repo is really awesome!

### ISSUE THREAD

**Issue by beetaa** [pikax/vue-composable#514]
By combining composition-api with the concept of PLUGIN, I can see a bright future with this new thinking and practicing and, the code base is quite clean, useful. thank you.

**Comment by blowsie** [comment]
I agree, and couldnt be more appreciative of the time and effort @pikax puts into this. 
Personally I just opted to [sponsor](https://github.com/sponsors/pikax) him. I hope the vue core team recognise his contributions and make him part of the team ;)

**Comment by pikax** [comment]
> By combining composition-api with the concept of PLUGIN, I can see a bright future with this new thinking and practicing and, the code base is quite clean, useful. thank you.

Thank you, composition-api opens the door for many reusable composables. 


> Personally I just opted to [sponsor](https://github.com/sponsors/pikax) him

Thank you so much @blowsie 💚 



> I hope the vue core team recognise his contributions and make him part of the team 😉

I'm currently part of the team :) My main focus is the composition-api plugin and making typescript better within vue ecosystem 💙
